Source/Filters/Video/YPVideoView.swift

```diff
@@ -112,8 +121,10 @@ extension YPVideoView {
 
         switch item.self {
         case let video as YPMediaVideo:
+            videoUrl = video.originalUrl
             player = AVPlayer(url: video.originalUrl)
         case let url as URL:
+            videoUrl = url
             player = AVPlayer(url: url)
         case let playerItem as AVPlayerItem:
             player = AVPlayer(playerItem: playerItem)
```

Comment:
In the last case:
```swift
case let playerItem as AVPlayerItem:
```
is it ever possible that the `asset` of `AVPlayerItem` is of type `AVURLAsset`? If it was would we also need to assign `videoUrl` here too? Or is in this path that is not possible?

Reply:
Hmm yeah I don't think this situation happens, but I added some code that sets it just in case.